C7/LogManager.cs

```diff
@@ -0,0 +1,68 @@
+using Godot;
+using Serilog;
+using Serilog.Templates;
+using Serilog.Templates.Themes;
+using Serilog.Core;
+using Serilog.Events;
+using Serilog.Configuration;
+using Serilog.Formatting;
+using System;
+using System.IO;
+
+public class LogManager
+{
+	static LogManager()
+	{
+		ExpressionTemplate consoleTemplate = new ExpressionTemplate(
+			"{@t:HH:mm:ss.fff} [{@l:u3}]{#if SourceContext is not null} {SourceContext}:{#end} {@m:lj}\n{#if @x is not null}\tException: {@x}{#end}",
```

Comment:
@QuintillusCFC commented: "Can we put a sample statement above the format, or an explanation of the more complex elements ([{@1:u3}] and {@m:lj}, IMO) to make it easier to read what the format is trying to do? Log formats are rarely intuitive and this one isn't too bad, but it's still handy to have the reference right there."